libs/designsystem/src/lib/directives/theme-color/theme-color.directive.ts

```diff
@@ -21,6 +21,12 @@ export class ThemeColorDirective {
     return this._isDefault;
   }
 
+  private _semiLight: boolean;
+  @HostBinding('class.semi-light')
+  get isSemiLight() {
+    return this._semiLight;
+  }
+
```

Comment:
This effectively also adds `semi-light` class for:
* `kirby-card`
* `kirby-badge`
* `kirby-icon`
* `kirby-progress-circle-ring`
* `kirby-toolbar`
* `kirby-modal-footer`
* `kirby-empty-state`

Is this intentionally?

Reply:
Yes, dont see any harm

Reply:
It might be - UX usually have a very strong opinion on what to use and when...